src/Angular-CSharp.csproj.in

```diff
@@ -46,14 +48,14 @@
     </Exec>
     <Error Condition="'$(ErrorCode)' != '0'" Text="Node.js is required to build and run this project. To continue, please install Node.js from https://nodejs.org/, and then restart your command prompt or IDE." />
     <Message Importance="high" Text="Restoring dependencies using 'npm'. This may take several minutes..." />
-    <Exec WorkingDirectory="$(SpaRoot)" Command="npm install" />
+    <Exec WorkingDirectory="$(SpaRoot)" Command="$(SpaInstallCommand)" />
```

Comment:
We probably want the same `Condition="'$(DOTNET_RUNNING_IN_CONTAINER)' != 'True'` here.

Reply:
Actually, this would need to go in the target I am not sure why the target pivots on `'$(Configuration)' == 'Debug'`

Reply:
> Actually, this would need to go in the target I am not sure why the target pivots on '$(Configuration)' == 'Debug'

There's two targets. `DebugEnsureNodeEnv` and `PublishRunWebpack`. The first is for development and has the Configuration condition. We care about skipping the spa build in the second.